### PR TITLE
Styled AvatarText

### PR DESCRIPTION
**Pull request by guilean**
**Checklist**
- [X] New feature (non-breaking change which adds functionality)

**Please check if the PR fulfills these requirements**
- [X] Tests for the changes have been added (for bug fixes / features)
- [X] Storybook or docs have been added / updated (for bug fixes / features)

**What is the ticket / issue associated with this change?** (Link to the ticket or issue)
https://trello.com/c/mMYjbJyj

### CODE REVIEW

File: packages/styled-components/src/components/primitives/Avatars/AvatarText/AvatarText.tsx [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+  switch (size) {
+    case Sizes.Large:
+    case Sizes.Medium:
+      return Typography.Types.Headline6;
+    case Sizes.Small:
+      return Typography.Types.Body;
+    case Sizes.ExtraSmall:
+      return Typography.Types.Eyebrow;
+    default:
+      return null;
```
Comment (tkanzakic) [inline packages/styled-components/src/components/primitives/Avatars/AvatarText/AvatarText.tsx:21]: is this necessary?

Comment (guilean) [reply packages/styled-components/src/components/primitives/Avatars/AvatarText/AvatarText.tsx]: Removed

**Review by tkanzakic — CHANGES_REQUESTED** [review]
missing export statement in the `/components/primitives/index.ts`

File: packages/styled-components/src/components/primitives/index.ts [hunk 2]
```diff
@@ -5,3 +5,4 @@ export * from './Link';
 export * from './Badges/NumericBadge';
 export * from './Typography';
 export * from './Video';
+export * from './Avatars/AvatarText';
```
Comment (tkanzakic) [inline packages/styled-components/src/components/primitives/index.ts:8]: please order alphabetically

**Review by tkanzakic — CHANGES_REQUESTED** [review]

**Review by tkanzakic — APPROVED** [review]

**Pull request merged**